app/views/dashboard/_status_chart_widget.html.erb

```diff
@@ -1,4 +1,4 @@
 <h4 class="header">
-  <%= _('Host Configuration Chart') %>
+  <%= title = _('Host Configuration Chart') %>
 </h4>
-<%= render_overview(@report, :class => 'statistics-pie small') %>
+<%= render_overview(@report, :class => 'statistics-pie small', :title => title) %>
```

Comment:
This looks like it's adding an additional tooltip to the chart, so now you get a tooltip per legend but also a tooltip over the top that's repeating what's in the widget title:

![screenshot from 2016-03-01 09-38-58](https://cloud.githubusercontent.com/assets/482089/13423219/7b6c03da-df91-11e5-9fef-c0076229c540.png)